business/services.go

```diff
@@ -214,9 +230,57 @@ func (in *SvcService) buildServiceList(namespace models.Namespace, svcs []core_v
 			Selector:               item.Spec.Selector,
 			IstioReferences:        svcReferences,
 			KialiWizard:            kialiWizard,
+			ServiceRegistry:        "Kubernetes",
 		}
 	}
 
+	// Add Istio Registry Services that are not present in the Kubernetes list
+	rSvcs = kubernetes.FilterRegistryServicesByServices(rSvcs, svcs)
```

Comment:
Here I see that Kiali first takes Services from K8 then takes the rest of services from Registry, right?
Wasn't the reason of introducing Registry in having listed from Registry first, then missing information from K8?

Reply:
Right.
The reationale for this was:
- Keep using the Kubernetes Services, as they information that is not collected in Istio (annotations, pod).
- Use the Istio registry for those services that are visible in the sidecars but not part of the kubernetes Services.

At least for the case of the Service list/details pages.

Probably we could introduce another use case for validations or similar when we want to perform a cross-namespace query with few data (exportTo, name, namespace, attribute, hostname).

Then, you're right, we could introduce a new flag in the business.SvcServiceCriteria that would help to use this case.